src/schematic/traits/volume_mounter.rs

```diff
@@ -101,28 +99,25 @@ impl VolumeMounter {
                 ..Default::default()
             }),
             ..Default::default()
-        }
+        })
     }
-    fn mount_policy(&self, volume: Option<&Volume>) -> String {
-        volume
-            .and_then(|vol| {
-                Some(match vol.access_mode {
-                    AccessMode::RO => "ReadOnlyMany",
-                    AccessMode::RW => match vol.sharing_policy {
-                        SharingPolicy::Shared => "ReadWriteMany",
-                        _ => "ReadWriteOnce",
-                    },
-                })
-            })
-            .unwrap_or("ReadWriteOnce")
-            .to_string()
+    fn mount_policy(&self, volume: &Volume) -> String {
+        match volume.access_mode {
+            AccessMode::RO => "ReadOnlyMany",
+            AccessMode::RW => match volume.sharing_policy {
+                SharingPolicy::Shared => "ReadWriteMany",
+                _ => "ReadWriteOnce",
+            },
+        }
+        .to_string()
     }
 
-    /// Locate the volume that this mounter is supposed to attach
+    /// Locate the volume that this mounter is supposed to attach that is not ephemeral
     fn find_volume(&self) -> Option<&Volume> {
         self.component.containers.iter().find_map(|c| {
             c.resources.volumes.as_ref().and_then(|vols| {
                 vols.iter()
+                    .filter(|v| !v.disk.as_ref().unwrap().ephemeral)
```

Comment:
Does it mean if we create a new PV, it's not ephemeral?

Reply:
that's the plan